### DISCUSSION TITLE
Trouble running WarsWorld pnpm dx command

### DISCUSSION THREAD

**Discussion by colorfularty** [WarsWorld/WarsWorld#90] [Q&A] [answered]
I'm following the README instructions trying to run  WarsWorld and when I reach step 5 on executing the pnpm dx command, I get the following error:

C:\Users\color\WarsWorld>pnpm dx

> @examples/next-websockets-starter@10.19.1 dx C:\Users\color\WarsWorld
> run-s db-up db-migrate-dev db-seed dev

> @examples/next-websockets-starter@10.19.1 db-up C:\Users\color\WarsWorld
> docker-compose up -d

Error response from daemon: Bad response from Docker engine
 ELIFECYCLE  Command failed with exit code 1.
ERROR: "db-up" exited with 1.
 ELIFECYCLE  Command failed with exit code 1.

I've Googled the issue, but no one is having the issue on Windows. I was wondering if someone else had run into this problem and had a solution or if I'm on my own here.

**Comment by FunctionDJ** [comment, marked as answer]
You need docker installed and the docker-compose command available. Check any docker tutorial on how to resolve your daemon issue. You can test more easily with the hello-world docker container (see Docker documentation).

**Reply by colorfularty** [reply]
Thanks for the quick reply! Yeah after scouring Youtube I found a guy with the same issue. Finally got it to work now!
